platform-sdk/docs/hashgraph_glossary.md

```diff
@@ -0,0 +1,250 @@
+# Hashgraph Glossary
+
+<a name="top"/>
+
+- [Rounds](#rounds)
+- [Fields of an event](#fields-of-an-event)
+- [Events](#events)
+- [Event relationships](#event-relationships)
+- [Generations](#generations)
+- [Address Books](#addressbooks)
+- [Parameters](#parameters)
+
+<a name="rounds"/>
+
+## Rounds
```

Comment:
should there be an intro explaining what rounds are?

Reply:
I agree.  Defining `round` by itself will also make the specialized terms `pending round` `consensus round` and others feel less circular.